### PR TITLE
New module added - Custom Default Ammo & update of DamageChange module

### PR DESCRIPTION
**Pull request by pandathebeasty**

### CODE REVIEW

**Comment by pandathebeasty** [comment]
@partiusfabaa Hi, can you check the plugin please?
I can't test it on my side

File: VIPCore/modules/VIP_CustomDefaultAmmo/VIP_CustomDefaultAmmo.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+public class CustomDefaultAmmo : VipFeatureBase
+{
+    public override string Feature => "CustomDefaultAmmo";
+    private CustomDefaultAmmoConfig _config;
+
+    public CustomDefaultAmmo(VipCustomDefaultAmmo vipCustomAmmo, IVipCoreApi api) : base(api)
+    {
+        _config = new CustomDefaultAmmoConfig();
```
Comment (partiusfabaa) [inline VIPCore/modules/VIP_CustomDefaultAmmo/VIP_CustomDefaultAmmo.cs:47]: you create a new instance of the `CustomDefaultAmmoConfig` class, but you dont load any configuration data into it. Shouldn't this be loaded from a configuration file?

Comment (pandathebeasty) [reply VIPCore/modules/VIP_CustomDefaultAmmo/VIP_CustomDefaultAmmo.cs:47]: Yes, indeed
But how I can do it in current context?

Comment (partiusfabaa) [reply VIPCore/modules/VIP_CustomDefaultAmmo/VIP_CustomDefaultAmmo.cs]: > Yes, indeed But how I can do it in current context?

well for example, `_config = LoadConfig<CustomDefaultAmmoConfig>("vip_customdefaultammo);`

Comment (pandathebeasty) [reply VIPCore/modules/VIP_CustomDefaultAmmo/VIP_CustomDefaultAmmo.cs]: @partiusfabaa can you check if what I've changed will work?

**Pull request merged**